CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # MicroProfile Language Server Changelog
 
+## [0.13.0](https://github.com/eclipse/lsp4mp/milestone/14?closed=1) (October 17, 2024)
```

Comment:
I would get rid of the 'feat', 'fix' pre-fixes.

Reply:
fixed